docs/submission/validating-molecular-data.md

```diff
@@ -0,0 +1,447 @@
+
+A description of all other payload sections is shown below: 
+
+|  Payload Field     | Payload Section |  Attribute                                                |  Description                                                                                                                                                                  |  Permissible Values     |  
+|-|-|-|-|-|
+|  studyId           |  |  ![Required](/assets/submission/dictionary-required.svg)  |  ARGO `Program ID`, the unique identifier of your program. If you have logged into the ARGO Data Platform, this is the Program ID that you see in the Program Services area.  |                         |  
+|  analysisType      |  |  ![Required](/assets/submission/dictionary-required.svg)  |  The type of molecular data that is being submitted in the payload.                                                                                                           |  sequencing_experiment  |  
+|  submitter_sequencing\_ experiment_id  | experiment |  ![Required](/assets/submission/dictionary-required.svg)  |  The unique identifier of the sequencing experiment.                                                     |                                    |  
+|  platform                              | experiment |  ![Required](/assets/submission/dictionary-required.svg)  |  The sequencing platform type used in data generation.                                                   | CAPILLARY, LS454, ILLUMINA, SOLID, HELICOS, IONTORRENT, ONT, PACBIO, Nanopore, BGI |  
+|  platform_model                        | experiment |                                                           |  The model number of the sequencing machine used in data generation.                                     |                                    |  
+|  experimental_strategy                 | experiment |  ![Required](/assets/submission/dictionary-required.svg)  |  The primary experimental method. For sequencing data it refers to how the sequencing library was made.  |  WGS, WXS, RNA-Seq, Bisulfite-Seq  |  
+|  sequencing_date                       | experiment |                                                           |  Date sequencing was performed.      |datetime format, for example: 2019-06-16,    2019-06-16T20:20:39+00:00  |  
+|  read_group_count  |  |  ![Required](/assets/submission/dictionary-required.svg)  |  The number of read groups in the molecular files being submitted.                                                                                                               |   |  
+| read_group_id_in_bam | read_groups |   |  Optional field indicating the read group id that is in the BAM.  This **cannot** be submitted for FASTQ files. |                                                                      |  
```

Comment:
should you say "optional field" in the description of other optional fields?

Reply:
i added a better descrption